lib/policy_machine_storage_adapters/in_memory.rb

```diff
@@ -28,6 +28,10 @@ class InMemory
       # The results are paginated via will_paginate using the pagination params in the params hash
       # The find is case insensitive to the conditions
       define_method("find_all_of_type_#{pe_type}") do |options = {}|
+        if options.delete(:where)
```

Comment:
So long as we limit the functionality, it'll require a finite amount of work to maintain feature parity across adapters. This is immediately useful in Dalton, since it uses two different adapters, and will also be helpful in the future when we (likely) move away from SQL in Dalton. If we allow custom `where`s now, then in a year that migration will be harder.